app/views/venues/index.html.erb

```diff
@@ -101,27 +101,26 @@
         <th>Neighborhood</th>
         <th>Actions</th>
       </tr>
-
-      <% @venues.each do |venue| %>
-      <tr>
-        <td><%= venue.name %></td>
-        <td><%= venue.address %></td>
-        <td>
-          <% if venue.neighborhood.present? %>
-            <a href="/neighborhoods/<%= venue.neighborhood.id %>">
-              <%= venue.neighborhood.name %>
-            </a>
-          <% end %>
-        </td>
-        <td>
-          <a href="/venues/<%= venue.id %>" class="btn btn-primary">Show</a>
-          <a href="/venues/<%= venue.id %>/edit" class="btn btn-warning">Edit</a>
-          <a href="/delete_venue/<%= venue.id %>" class="btn btn-danger" rel="nofollow">Delete</a>
-        </td>
-      </tr>
+      
+      <% current_user.bookmarked_venues.each do |bookmark_venue| %>
```

Comment:
Should probably have the iterated variable be "venue" for readability, but it really doesn't matter. Will commit this now.